achievements/tomcat.md

```diff
@@ -2,7 +2,17 @@
 
 Prior to the Hackathon the Tomcat commiters created a [wiki page](https://cwiki.apache.org/confluence/display/TOMCAT/EU+FOSSA+May+2019) with topics to work on and discuss.
 
-Most of the people at the Tomcat table got interested in getting Tomcat to run on GraalVM. 
+## GraalVM
+Some of the people at the Tomcat table got interested in getting Tomcat to run on GraalVM. 
 Because we already knew that Tomcat would not run with the latest published version of GraalVM, we started with compiling the master branch of GraalVM from Github. There is a commit in master just done a few days before the event that should fix some issues that appeared before.
 
-Compiling GraalVM from source turned out to be a little bit tricky, because it did not compile with a regular version of the Java VM but required a special JVM.
\ No newline at end of file
+Compiling GraalVM from source turned out to be a little bit tricky, because it did not compile with a regular version of the Java VM but required a special JVM.
+
+## Community and Builds
+The team instructed a few attendees on how to build Apache Tomcat from source either from checkout-oput the code from git or from a release package. Also gave instructions for forking the project on GitHub for future hacking and pull-requests.
+
+## Fixes
+
+One enhancement request was completed fixed.
+
+One new bug was filed.
```

Comment:
"completed fixed" ?
Could you use a list ? Thanks.

Reply:
Yes, "completed fixed", as opposed to "completed rejected" or some other kind of resolution. Feel free to change formatting. This page is for *your* group's achievements, not the Tomcat project's. We already have separate tracking mechanisms, etc.